python/az/aro/azext_aro/_aad.py

```diff
@@ -51,6 +54,12 @@ def create_application(self, display_name):
 
         return app, password
 
+    def set_managed_app_url(self, cli_ctx):
+        cloud_name = get_active_cloud_name(cli_ctx)
+        if cloud_name not in ["AzureCloud", "AzureUSGovernment"]:
```

Comment:
So we don't have to keep track of cloud names in multiple places...

```suggestion
        if cloud_name not in self.MANAGED_APP.keys():
```